pagefaultgames/pokerogue
# [QoL] Improve Input Accuracy by Refactoring Button Handling

## What are the changes?
The changes involve a refactor in the inputs-controller & touch-controls to manage holding buttons, which also fixes the issue of double inputs.

## Screenshots
![image](https://github.com/pagefaultgames/pokerogue/assets/44787002/765397f7-ec0c-4c81-ac11-3bdd9aab0d85)

## How to test the changes?
To test the changes:
1. Check out the branch with the changes.
2. Test pressing a button to ensure that only one input is registered.
3. Test holding a button to ensure that input is repeated.
4. Run `npm run test` to ensure all unit tests are passing.

## Checklist
